coreTable/OwidTableUtil.ts

```diff
@@ -0,0 +1,10 @@
+import { ColumnSlug } from "./CoreTableConstants"
+import { OwidColumnDef, OwidTableSlugs } from "./OwidTableConstants"
+
+export function timeColumnSlugFromColumnDef(def: OwidColumnDef) {
+    return def.isDailyMeasurement ? OwidTableSlugs.day : OwidTableSlugs.year
+}
+
+export function makeOriginalTimeSlugFromColumnSlug(slug: ColumnSlug) {
```

Comment:
nits:

Arrow functions? Reduce your code golf scores by 66% :)

```typescript
export const makeOriginalTimeSlugFromColumnSlug = (slug: ColumnSlug) => `${slug}-originalTime`
```

Reply:
arrows are for arqueros